src/vm/perfinfo.cpp

```diff
@@ -41,8 +41,21 @@ void PerfInfo::LogImage(PEFile* pFile, WCHAR* guid)
 
     SString value;
     const SString& path = pFile->GetPath();
-    PEImageLayout *pLoadedLayout = pFile->GetLoaded();
-    SIZE_T baseAddr = (SIZE_T)pLoadedLayout->GetBase();
+    if (path.IsEmpty())
```

Comment:
Why do we filter empty paths?

Reply:
If the path is empty, then we have no way of using this information to resolve symbols.  Instead, the symbols will be resolved when the method is jitted via perf-<pid>.map.